components/html-block/README.md

```diff
@@ -109,26 +109,25 @@ Examples are provided to display how user-authored math can be embedded within y
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/html-block/html-block.js';
-  import '@brightspace-ui/core/components/icons/icon.js';
 
-</script>
-<script>
-	window.D2L = {};
-	D2L.LP = {};
-	D2L.LP.Web = {};
-	D2L.LP.Web.UI = {};
-	D2L.LP.Web.UI.Flags = {
-		Flag: (feature, defaultValue) => {
-			if (feature === 'us125413-mathjax-render-latex') return true;
-			else return defaultValue;
-		}
-	};
+  window.D2L = {};
+  D2L.LP = {};
+  D2L.LP.Web = {};
+  D2L.LP.Web.UI = {};
+  D2L.LP.Web.UI.Flags = {
+    Flag: (feature, defaultValue) => {
+      if (feature === 'us125413-mathjax-render-latex') return true;
+      else return defaultValue;
+    }
+  };
+  <!-- docs: start hidden content -->
+  document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({ renderLatex: true });<!-- docs: end hidden content -->
```

Comment:
This demo didn't seem to show the Latex rendered properly so I used this to do so